### PR TITLE
feat: add javascript jwt support

### PR DESCRIPTION
**Pull request by vjerci**
## Description
This pr cleans up a little bit previously added js rules
This pr adds a rule for javascript, specifically supporting detections of encoding sensitive data in jwt.

It searches for this pattern:
```javascript
jwt.sign($<DATA_TYPE>)
```

## Related
Solves a part of https://github.com/Bearer/curio/issues/504

## Checklist

- [x] I've added test coverage that shows my fix or feature works as expected.
- [x] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [x] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: pkg/commands/process/settings/rules/javascript/lang/jwt.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ✅ If you need to store user's information, use their unique database identifier instead of personal identifiable information:
+
+    ```javascript
+      const jwt = require('jsonwebtoken');
+      const token = jwt.sign({ user: user.uuid });
+    ```
+
+    <!--
+    ## Resources
+    Coming soon.
```
Comment (cfabianski) [inline pkg/commands/process/settings/rules/javascript/lang/jwt.yml:43]: @markmichon happy to have you review this new one :)

File: pkg/commands/process/settings/rules/javascript/lang/weak_encryption.yml [hunk 2]
```diff
@@ -35,10 +35,13 @@ metadata:
   remediation_message: |
     ## Description
 
-    Sensitive data should be encrypted with strong encryption algorithms
+    Sensitive data should be encrypted with strong encryption algorithms like aes-256-cbc
```
Comment (cfabianski) [inline pkg/commands/process/settings/rules/javascript/lang/weak_encryption.yml:38]: cc @markmichon

**Review by cfabianski — APPROVED** [review]

File: pkg/commands/process/settings/rules/javascript/lang/jwt.yml (hunk 1 above)
Comment (vjerci) [reply pkg/commands/process/settings/rules/javascript/lang/jwt.yml:43]: resolving this for now so I can merge it, lets open a separate pr for any javascript docs related changes.

File: pkg/commands/process/settings/rules/javascript/lang/weak_encryption.yml (hunk 2 above)
Comment (vjerci) [reply pkg/commands/process/settings/rules/javascript/lang/weak_encryption.yml:38]: resolving this for now so I can merge it, lets open a separate pr for any javascript docs related changes.

**Pull request merged**